### PR TITLE
added tests for delete API

### PR DESCRIPTION
**Pull request by jnv27**
Hey @Manvityagi @abdus , I have added tests for the delete API as per issue #94 , please check and merge it.

### CODE REVIEW

**Review by abdus — APPROVED** [review]
appreciate the contribution. I think you could have added more test cases .. maybe cover a few more edge cases. 
ready to merge once @Manvityagi approves.

File: tests/managers/opportunity-test.js
Comment (Manvityagi) [inline tests/managers/opportunity-test.js:159]: I don't understand what have you tried to achieve with this test case? 
Can you elaborate, how it is assuring `should not delete if Opportunity does not exist`?

Comment (jnv27) [reply tests/managers/opportunity-test.js:159]: @Manvityagi , in the line 156 we are finding the data that we have to delete in DB by the id, now in the 'it' block written in line 160, there could be two possibilities, first when we couldn't find any data in DB using that Id, in that case the value of deletedOpportunity would be null and hence will return false in the in the line 162. Now if this case is passed then, for the confirmation I'm checking if the deletedOpportunity._id is equal to stubValue._id, which also should be true if the data which we want to delete is present in the DB, and if it isn't present then in that case both the cases will fail and hence it won't delete anything

**Pull request merged**
